migrations/versions/2025_03_06_1130-769f09b6d992_add_provider_endpoint_fields_to_muxes.py

```diff
@@ -0,0 +1,133 @@
+"""add provider endpoint fields to muxes
+
+Revision ID: 769f09b6d992
+Revises: e4c05d7591a8
+Create Date: 2025-03-06 11:30:11.647216+00:00
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "769f09b6d992"
+down_revision: Union[str, None] = "e4c05d7591a8"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # Begin transaction
+    op.execute("BEGIN TRANSACTION;")
+
+    # Add the new columns
+    op.execute(
+        """
+        ALTER TABLE muxes
+        ADD COLUMN provider_endpoint_type TEXT;
+        """
+    )
+    op.execute(
+        """
+        ALTER TABLE muxes
+        ADD COLUMN provider_endpoint_name TEXT;
+        """
+    )
```

Comment:
Do we need these new references in the table? We could probably first just query the providers table to get the ID and then keep using the database schema as-is.